TraceInfrastructure/Backend/BackendTrace.c

```diff
@@ -151,10 +151,10 @@ void LoadDump(void *address)
 void DumpLoadAddrValue(void *MemValue, int size)
 {
```

Comment:
This function should only handle the value, not the address. If you need both you should call both.

I also don't see why you renamed it from LoadAddress. Ideally, your code should be able to coexist with mine with no real change.